tests/check/check_test.go

```diff
@@ -60,26 +54,42 @@ func testRunTestMatrix(t *testing.T, engine string, experimental bool) {
 
 		conn := testutils.CreateGrpcConnection(t, cfg.GRPC.Addr)
 
-		runTestMatrixSuite(t, openfgav1.NewOpenFGAServiceClient(conn))
+		RunTestMatrixSuite(t, openfgav1.NewOpenFGAServiceClient(conn))
 	})
 }
 
 func TestCheckMemory(t *testing.T) {
 	testRunAll(t, "memory")
 }
 
+func TestMatrixMemory(t *testing.T) {
```

Comment:
Why not put it in `TestCheckMemory`?

Reply:
unfortunately the goleak.VerifyNone does not play nicely with the testRunAll :(